LEAF_Request_Portal/enqueue/QueueConfig.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class QueueConfig
+{
+    public $key = 'ENTER_AWS_KEY';
```

Comment:
This configuration should be pulled from environment variable.  See globals.php for pattern.

For local development, put place-holder in `.env` file and `docker-compose.yml`